CHANGELOG.md

```diff
@@ -54,6 +54,8 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
   </script/>
 
   <div></div>
+
+- Add lint rule useJsxKeyInIterable from Eslint rule [`react/jsx-key`](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-key.md)
```

Comment:
This one should be moved out of the backtick quotes.

Reply:
@vohoanglong0107 could you move your changelog entry one more line down, so the three backticks (`) end the code snippet before your entry?

Reply:
Aaaaaaaaaa I see. I forgot to reset the conceallevel of vim so I couldn't see the backtick